lib/config.go

```diff
@@ -173,6 +173,9 @@ type RuntimeConfig struct {
 	// Will also set the readonly flag in the OCI Runtime Spec.  In this mode containers
 	// will only be able to write to volumes mounted into them
 	ReadOnly bool `toml:"read_only"`
+
+	// If specified the prefix is added to the source for every bind mount
```

Comment:
You don't need “If specified” here, because the default empty string is a valid, no-op prefix.  I recommend copy/pasting the `crio.8.md` description in here.  Or maybe:

```go
// BindMountPrefix is the prefix to use for the source of the bind mounts.
```

depending on whether [Godocs'][1]

> Notice this comment is a complete sentence that begins with the name of the element it describes.

applies to these structure properties or not.

[1]: https://blog.golang.org/godoc-documenting-go-code